frontend/src/pages/modelServing/screens/projects/ModelServingPlatform.tsx

```diff
@@ -0,0 +1,156 @@
+import ManageServingRuntimeModal from './ServingRuntimeModal/ManageServingRuntimeModal';
+import ModelMeshServingRuntimeTable from './ModelMeshSection/ServingRuntimeTable';
+import ModelServingPlatformButtonAction from './ModelServingPlatformButtonAction';
+
+const ModelServingPlatform: React.FC = () => {
+  const {
+    dashboardConfig: {
+      spec: {
+        dashboardConfig: { disableKServe, disableModelMesh },
+      },
+    },
+  } = useAppContext();
+  const [isOpen, setOpen] = React.useState(false);
+  const [platformSelection, setPlatformSelection] = React.useState<ServingRuntimePlatform>();
+
+  const {
+    servingRuntimes: {
+      data: servingRuntimes,
+      loaded: servingRuntimesLoaded,
+      error: servingRuntimeError,
+      refresh: refreshServingRuntime,
+    },
+    servingRuntimeTemplates: { data: templates, loaded: templatesLoaded, error: templateError },
+    servingRuntimeTemplateOrder: { data: templateOrder },
+    servingRuntimeTemplateDisablement: { data: templateDisablement },
+    serverSecrets: { refresh: refreshTokens },
+    inferenceServices: { refresh: refreshInferenceServices },
+    currentProject,
+  } = React.useContext(ProjectDetailsContext);
+
+  const { refresh: refreshAllProjects } = React.useContext(ProjectsContext);
+
+  const templatesSorted = getSortedTemplates(templates, templateOrder);
+  const templatesEnabled = templatesSorted.filter((template) =>
+    getTemplateEnabled(template, templateDisablement),
+  );
+
+  const emptyTemplates = templatesEnabled.length === 0;
+  const emptyModelServer = servingRuntimes.length === 0;
+
+  const currentProjectServingPlatform = getProjectModelServingPlatform(
+    currentProject,
+    disableKServe,
+    disableModelMesh,
+  );
+
+  const isModelMesh =
+    currentProjectServingPlatform === ServingRuntimePlatform.MULTI ||
+    platformSelection === ServingRuntimePlatform.MULTI;
+
+  const shouldShowPlatformSelection =
+    !disableKServe && !disableModelMesh && !currentProjectServingPlatform;
+
+  const initialState = !platformSelection && !currentProjectServingPlatform;
+
+  return (
+    <>
+      <DetailsSection
+        id={ProjectSectionID.MODEL_SERVER}
+        title={ProjectSectionTitles[ProjectSectionID.MODEL_SERVER]}
+        actions={
+          initialState
+            ? undefined
+            : [
+                <ModelServingPlatformButtonAction
+                  tooltipContent={
+                    // show tooltip for deploying a model
+                    emptyTemplates
+                      ? `At least one serving runtime must be enabled to ${
+                          isModelMesh ? 'add a model server' : 'deploy a model'
+                        }. Contact your administrator.`
+                      : isModelMesh
+                      ? 'A model server specifies resources available for use by one or more supported models, and includes a serving runtime.'
+                      : ''
```

Comment:
This is the thing that @kaedward or @vconzola can help us 😅

Reply:
@lucferbux I already removed the tooltip for KServe button based on the feedback here https://redhat-internal.slack.com/archives/C05SMJ09DD2/p1698282299029969?thread_ts=1698247959.406999&cid=C05SMJ09DD2